hangups/ui/__main__.py

```diff
@@ -198,11 +198,23 @@ def _on_event(self, conv_event):
 
     def _on_quit(self):
         """Handle the user quitting the application."""
-        future = asyncio.async(self._client.disconnect())
-        future.add_done_callback(lambda future: future.result())
+        asyncio.ensure_future(self._client.disconnect())
 
 
-class LoadingWidget(urwid.WidgetWrap):
+class WidgetBase(urwid.WidgetWrap):
```

Comment:
I explained that in the comment of https://github.com/tdryer/hangups/pull/343/commits/d864d998ebd2b1b5c188152d25e49debb2134155

> The new class ``WidgetBase`` overrides the property definition for the method ``keypress`` in ``urwid.WidgetWrap``.
Using one method that overrides the property saves many pylint suppressions.

It also can be used to refactor the key-mapping in one place.